droidlet/tools/crowdsourcing/droidlet_static_html_task/issue_bonus.py

```diff
@@ -67,7 +67,11 @@ def issue_bonuses(task_name: str) -> list:
                         amount = float(f'{(click["id"]["interactionScores"]["stoplight"] * 0.30):.2f}')
                         total_bonus += amount
                         new_bonus_records.append((task_name, unit_id, worker.worker_name, amount))
-                        bonus_result, _ = worker.bonus_worker(amount, "Virtual assistant interaction quality bonus", unit)
+                        bonus_result = False
+                        try:
+                            bonus_result, _ = worker.bonus_worker(amount, "Virtual assistant interaction quality bonus", unit)
+                        except:
```

Comment:
what would be the possible reasons bonus_worker fail here? Should we handle/log failed cases?

Reply:
I'm not sure I have a complete understanding of the reasons it could fail, but the one time it has failed so far it's because AWS has no record of the HIT happening.